ui-tests/src/test/java/io/syndesis/qe/steps/integrations/IntegrationSteps.java

```diff
@@ -334,34 +334,34 @@ public void sheAddsSecond() throws Throwable {
 	}
 
 	@And("^sets jms subscribe inputs source data$")
-	public void setJmsSubscribeData(DataTable sourceMappingData) {
+	public void setJmsSubscribeData(DataTable sourceMappingData) throws IllegalArgumentException, IllegalAccessException {
 		for (Map<String, String> source : sourceMappingData.asMaps(String.class, String.class)) {
 			for (String field : source.keySet()) {
-				String tagType = editPage.getJmsSubscribeComponent().checkAndGetFieldType(field);
-				assertThat(tagType, notNullValue());
-				editPage.getJmsSubscribeComponent().fillInput(field, source.get(field), tagType);
+				SelenideElement element = editPage.getJmsSubscribeComponent().checkAndGetFieldType(field);
+				assertThat(element, notNullValue());
+				editPage.getJmsSubscribeComponent().setElementValue(source.get(field), element);
 			}
 		}
 	}
 
 	@And("^sets jms request inputs source data$")
-	public void setJmsRequestData(DataTable sourceMappingData) {
+	public void setJmsRequestData(DataTable sourceMappingData) throws IllegalArgumentException, IllegalAccessException {
```

Comment:
I hope I removed all of those exception in my PR. Method declaration should be exception-less :) Pls double-check and eventually switch any exception to assert rather.